windows/introduction.tw2

```diff
@@ -1,8 +1,8 @@
 ::Windows Introduction
 # Installing Exercism on Windows
 
-Do you know whether you are using Windows 10?
+Are you using Windows 10?
 
-- [[I am using Windows 10->Download and unzip 64-bit version on Windows 10]]
-- [[I am not using Windows 10->Talk to a Volunteer]]
-- [[I am not sure what verion of Windows I am using->Finding out what version of Windows you are using]]
+- [[Yes, I am using Windows 10.->Download and unzip 64-bit version on Windows 10]]
+- [[No, I am not using Windows 10.->Talk to a Volunteer]]
+- [[Assistance please, I am not sure what version of Windows I am using.->Finding out what version of Windows you are using]]
```

Comment:
I'm not a huge fan of the "Assistance please," here. In conversation, if you were walking around the room while people were getting set up for a workshop and asked "Are you using Windows 10?" they wouldn't respond "Assistance please, ..." they'd respond "I'm not sure..." perhaps with the qualification "I'm not sure what version of Windows I am using."